examples/pytorch/graphsage/main.py

```diff
@@ -19,7 +19,7 @@
     SamplingStrategy,
     GraphEngineBackend,
 )
-from model import SupervisedGraphSage, UnSupervisedGraphSage
+from model import SupervisedGraphSage, UnSupervisedGraphSage  # type: ignore
```

Comment:
why do we need to ignore types in imports?

Reply:
Mypy does not handle the repeated filenames and it is hard to get the imports right for each of the examples. The most important thing is how the examples use the trainer and GE to verify those types, not so much the model implementation is typed right.